### ISSUE TITLE
phpDoc 3 stops at "Execute transformation using writer "twig""

### ISSUE THREAD

**Issue by filips123** [phpDocumentor/phpDocumentor2#1995]
When I start phpDocumentor 3, it stops at "Execute transformation using writer "twig"" even if there are only few files to parse.

## Expected Behavior

phpDocumentor 3 should parse and build documentation correctly.

## Actual Behavior

When I start phpDocumentor 3, it stops at "Execute transformation using writer "twig"" even if there are only few files to parse.
Output from phpDocumentor is:
```
15:30:21 INFO      [app] Starting to parse file: wp-config-sample.php [] []
15:30:21 INFO      [app] Parsing wp-config-sample.php [] []
15:30:21 INFO      [app] Starting to parse file: wp-settings.php [] []
15:30:21 INFO      [app] Parsing wp-settings.php [] []
15:30:22 INFO      [app] Elapsed time to parse all files: 224.6s [] []
15:30:22 INFO      [app] Peak memory usage: 60M [] []
15:30:24 INFO      [app]
Applying 17 transformations [] []
15:30:24 INFO      [app]   Initialize writer "phpDocumentor\Plugin\Core\Transformer\Writer\FileIo" [] []
15:30:24 INFO      [app]   Initialize writer "phpDocumentor\Plugin\Twig\Writer\Twig" [] []
15:30:24 INFO      [app]   Initialize writer "phpDocumentor\Plugin\Graphs\Writer\Graph" [] []
15:30:24 INFO      [app]   Writer FileIo  using query "copy" on .htaccess [] []
15:30:24 INFO      [app]   Execute transformation using writer "FileIo" [] []
15:30:24 INFO      [app]   Writer FileIo  using query "copy" on images [] []
15:30:24 INFO      [app]   Execute transformation using writer "FileIo" [] []
15:30:24 INFO      [app]   Writer FileIo  using query "copy" on css [] []
15:30:24 INFO      [app]   Execute transformation using writer "FileIo" [] []
15:30:24 INFO      [app]   Writer FileIo  using query "copy" on js [] []
15:30:24 INFO      [app]   Execute transformation using writer "FileIo" [] []
15:30:24 INFO      [app]   Writer FileIo  using query "copy" on font [] []
15:30:24 INFO      [app]   Execute transformation using writer "FileIo" [] []
15:30:24 INFO      [app]   Writer twig  using query "namespace" on index.html [] []
15:30:24 INFO      [app]   Execute transformation using writer "twig" [] []
15:30:24 INFO      [app]   Writer twig  using query "indexes.namespaces" on  [] []
15:30:24 INFO      [app]   Execute transformation using writer "twig" [] []
15:31:15 INFO      [app]   Writer twig  using query "indexes.classes" on  [] []
15:31:15 INFO      [app]   Execute transformation using writer "twig" [] []
```

If I set target directory, documentation will be there, but there will also be some other classes and namespaces. I assume they are from phpDocumentor.

## Steps to Reproduce the Problem

  1. Create few files with docblocks. I am using [`wp-config-sample.php`](https://github.com/WordPress/WordPress/blob/master/wp-config-sample.php) and [`wp-settings.php`](https://github.com/WordPress/WordPress/blob/master/wp-settings.php) from WordPress.
  2. Run phpDocumentor with set target dir (`php phpDocumentor.phar run -d . -t docs`)
  3. Watch output and target folder

## Your environment

* Version used: 3.0.0-alpha.2
* Install method: PHAR
* php version: 7.2.6
* Operating system and versio: Ubuntu 16.04
* Hosting type: [Codenvy](https://codenvy.io) PHP stack

## Screenshots

**Output from phpDocumentor:**
![output](https://user-images.githubusercontent.com/16626308/41722626-620a81da-7569-11e8-9125-9369cce6210c.png)

**Builded documentation:**
![documentation](https://user-images.githubusercontent.com/16626308/41722661-78ed1b56-7569-11e8-8ad1-3deac9b46613.png)

**Comment by filips123** [comment]
**Update:** After about 10 minutes of waiting, the command was finished, but the documentation is still wrong.

**Output (I split it into two images and this is complete output):**
![output1](https://user-images.githubusercontent.com/16626308/41723597-cc195ca2-756b-11e8-8595-8c9b46bebf2c.png)
![output2](https://user-images.githubusercontent.com/16626308/41723645-e00f24e4-756b-11e8-9a36-41f4308170dc.png)

**Builded documentation:**
![documentation](https://user-images.githubusercontent.com/16626308/41723714-14fe2cfe-756c-11e8-8946-539f5aad2a96.png)

**Comment by jaapio** [comment]
Thanks for your extended report! 
We have an issue with the `-d` flag in phpdoc3. It is already fixed but some other issues need to be solved before we can release a new phar. Please follow #1960 for more updates. 
Closing this one because it is a duplicate of the issues reported there.

The long processing time is most likely because of the large content in the working directory?

**Comment by filips123** [comment]
> The long processing time is most likely because of the large content in the working directory?
I don't have large content in the working directory. There are only two files (`wp-config-sample.php` and `wp-settings.php` from WordPress).

**Comment by mvriel** [comment]
> Also, will phpDocumentor 3 support DocBlock templates? There is another issue #42, but is very old.

DocBlock template support is still desired (AFAIC) but is not simple to achieve. Due to focus and time constraints I cannot make any promises if and when this feature shall land.

**Comment by jaapio** [comment]
I think this was a bug in an alpha version. I cannot reproduce slow behavior. So closing this one.
